src/utils.js

```diff
@@ -52,13 +52,13 @@ const generatorPrompt = (projectName) => {
       choices: ["React", "No framework"],
       result(value) {
         let parsed = value.toLowerCase();
-        if (parsed == "no framework") {
+        if (parsed === "no framework") {
           parsed = "vanilla";
         }
         return parsed;
       },
     },
-  ]).catch((e) => {
+  ]).catch((_e) => {
```

Comment:
Below it says see other log lines for details. If we ignore the error will we see the details? Should we console log the error?

Reply:
I'll add logging of the error anyway. All I did was the `_` for the lint report